packages/backend/src/tokens/jwt/signJwt.test.ts

```diff
@@ -0,0 +1,36 @@
+import type QUnit from 'qunit';
+
+import {
+  mockJwtHeader,
+  mockJwtPayload,
+  pemEncodedPublicKey,
+  pemEncodedSignKey,
+  publicJwks,
+  signingJwks,
+} from '../fixtures';
+import { __unstable__signJwt } from './signJwt';
+import { decodeJwt, hasValidSignature } from './verifyJwt';
+
+export default (QUnit: QUnit) => {
+  const { module, test } = QUnit;
+
+  module('signJwt(payload, options)', () => {
+    test('signs a JWT with a JWK formatted secret', async assert => {
+      const jwt = await __unstable__signJwt(mockJwtPayload, signingJwks, {
+        algorithm: mockJwtHeader.alg,
+        header: mockJwtHeader,
+      });
+
+      assert.true(await hasValidSignature(decodeJwt(jwt), publicJwks));
```

Comment:
🙃 Isn't it more straightforward if you use `verifyJwt`?

Reply:
Yes! Initially I didn't update verifyJWT so I used its parts. Will change.

Reply:
Changed https://github.com/clerkinc/javascript/pull/1786/commits/c21e08a5c4363f2be544af4945a040a07f114f08